tests/check_consistent.py

```diff
@@ -220,8 +221,9 @@ def get_precommit_requirements() -> dict[str, SpecifierSet]:
         if not repo.get("python_requirement", True):
             continue
         hook = repo["hooks"][0]
-        package_name, package_rev = hook["id"], repo["rev"]
-        package_specifier = SpecifierSet(f"=={package_rev.removeprefix('v')}")
+        package_name = Path(urllib.parse.urlparse(repo["repo"]).path).name
```

Comment:
Is this meant to be part of this PR?

Reply:
Yes. The test as it currently stands is poorly designed. Without this change, `check_consistent.py` fails if we add `pre-commit-hooks` to our pre-commit config, because of the fact that the hooks in the `pre-commit-hooks` repo have different names to the repo as a whole.

Reply:
Example failing run: https://github.com/python/typeshed/actions/runs/3949452817/jobs/6760681897

Reply:
I see, thanks!